md/deploy-bundle.md

```diff
@@ -12,6 +12,12 @@ Thus, for installations using native distribution packages (that more often than
 it would be up to you to adapt the documented steps to your very own folder layout.
 :::
 
+:::warning
+Remember that the recommended way of installing Bonita is to use the provided [Tomcat](tomcat-bundle.md) / [WildFly](wildfly-bundle.md) bundle.  
```

Comment:
Should we mention WildFly here, It is not recommended anymore since it will be deprecated.

Reply:
I let you add the `deprecated` word near each `wildfly` word on the PR you will do to deprecate the WildFly bundle.
What do you think?

Reply:
ok 👍